tools.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
+
+package core
+
+import (
+	_ "github.com/smola/gocompat"
```

Comment:
I read the usage instructions on the `gocompat` repo, but I'm not sure I follow why we need an import for side-effects here. Isn't the CLI tool self-contained?

Reply:
It's how one tricks `go mod` into tracking it. That way, one can run `go run github.com/smola/gocompat/cmd/gocompat` and always get the right version.

Reply:
The build that is made such that it isn't used for the build.

Reply:
What a hack... *sigh* `scope=test` in Maven, `devDependencies` in npm.